### PR TITLE
Solves issue #106.

### PR DESCRIPTION
**Pull request by lelekaz**

### CODE REVIEW

**Comment by paytonshaltis** [comment]
Looks good and works on my end! Just to confirm, the 'all' button is meant to *toggle* all of the switches, not to turn all switches on, correct?

**Comment by lelekaz** [comment]
> Looks good and works on my end! Just to confirm, the 'all' button is meant to _toggle_ all of the switches, not to turn all switches on, correct?

That's what I coded it to do, but if anyone wants it coded differently let me know! :)

**Comment by sderacy** [comment]
> Just to confirm, the 'all' button is meant to _toggle_ all of the switches, not to turn all switches on, correct?
I see what you're saying.

On one hand, I could see someone saying, "Wouldn't it be more confusing to the user if they clicked "all"? If they were trying to grab all of the social media handles, and instead it turned some on and turned some off?"

But I actually really like this implementation because if you had 20 social media handles, I'd much rather a button that would take me 2 clicks to toggle all of them, than one button that toggles them on but can't toggle them off.

That being said, if you wanted to make a "smart" toggle, I think that'd be cleaner but that's a lot of attention to detail that may not be needed.

For elaboration on what I mean by a "smart" toggle, it would work something like this:
1. When clicked, it only toggles all social media accounts "on" if there's at least ONE social media account that is toggled on
2. Otherwise, turn them all off

TL;DR You really don't have to do it if you don't want to but that'd make the most sense I think.

-- More unneeded info --
Bonus points if you were trying to make it more optimal than just looking through the list of social media accounts in the array to see which are toggled, you could do something like keeping a count of all currently toggled switches instead of an array since you don't care which accounts are toggled, just that they are.

It'd work something like adding to a variable "numAccountsToggled" whenever someone toggles an account, and subtracting from that variable whenever someone toggles the account off. Then you'd just be able to check if that variable is 0 to see if all of the accounts are toggled off.

**Review by sderacy — APPROVED** [review]
Nice job!

**Comment by thadakn1** [comment]
just saw the latest commit! great work @lelekaz !!
